llvm/test/CodeGen/RISCV/rvv/fixed-vectors-vwsll.ll

```diff
@@ -499,6 +497,27 @@ define <16 x i16> @vwsll_vv_v16i16_zext(<16 x i8> %a, <16 x i8> %b) {
 }
 
 define <16 x i16> @vwsll_vx_i64_v16i16(<16 x i8> %a, i64 %b) {
+; CHECK-ZVBB32-LABEL: vwsll_vx_i64_v16i16:
```

Comment:
Why were there no CHECK lines of any kind previously?

Reply:
Committed 89ebb56152192e8ad535ddd11ae0f60334fd748a to resolve the conflict

Reply:
I missed that the rv32 and rv64 cases differed when precommitting these tests. Will rebase on 89ebb56152192e8ad535ddd11ae0f60334fd748a, thanks for splitting that out